### PR TITLE
test: add test-docker rule

### PR DESCRIPTION
**Pull request by beebzz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+ENV PATH="vendor/bin:$PATH"
 
 COPY src src
+COPY tests tests
 COPY composer* ./
 
-RUN curl --silent --show-error https://getcomposer.org/installer | php
-RUN mv composer.phar /usr/local/bin/composer
-RUN composer install --no-dev
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
```
Comment (childish-sambino) [inline Dockerfile:13]: Why this change instead of using curl?

Comment (beebzz) [reply Dockerfile:13]: Figured it was more of a best practice to use the proper composer image rather than curl it

**Review by childish-sambino — APPROVED** [review]
nit: should be a `test:` change
